src/Card/Card.component.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react"
+import styled, { css } from "styled-components"
+import CardBody, { CardBodyProps } from "./CardBody.component"
+import CardFooter, { CardFooterProps } from "./CardFooter.component"
+import CardHeader, { CardHeaderProps } from "./CardHeader.component"
+import CardImage, { ImageProps } from "./CardImage.component"
+import CardFlipper, { CardFlipperProps } from "./CardFlipper.component"
+
+interface CardComposition {
+  Body: React.FC<CardBodyProps>
+  Header: React.FC<CardHeaderProps>
+  Footer: React.FC<CardFooterProps>
+  Image: React.FC<ImageProps>
+  Flipper: React.FC<CardFlipperProps>
+}
+
+export interface CardProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Card will expand across available width
+   * @default false
+   */
+  fluid?: boolean
+  /**
+   * Raises the card, applying a small box-shadow
+   * @default false
+   */
+  raised?: boolean
+  /**
+   * Main image to be displayed
+   *
+   */
+  image?: string
+  /**
+   * Translate Y on hover
+   * @default false
+   */
+  hoverable?: boolean
+}
+
+const CardContainer = styled.div<CardProps>`
+  display: flex;
+  flex-direction: column;
+  border: 1px solid rgba(0, 0, 0, 0.1);
+  background: #fff;
+  border-radius: 5px;
+  overflow: hidden;
+  box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.1);
+  padding: 0;
+  width: 290px;
+  position: relative;
+  margin: 1m 0em;
+  justify-content: space-evenly;
+
+  & > a {
+    cursor: pointer;
+  }
+
+  // Modifiers
+  ${({ raised }) =>
+    raised &&
+    css`
+      box-shadhow: 4px 4px 10px rgba(0, 0, 0, 0.2);
+    `};
+
+  ${({ fluid }) =>
+    fluid &&
+    css`
+      width: 100%;
+      max-width: 9999px;
+    `}
+
+  ${({ hoverable }) =>
+    hoverable &&
+    css`
+      transform: translateY(0);
+      transition: transform 0.5s, box-shadow 0.5s;
+
+      &:hover {
+        transform: translateY(-10px);
+        box-shadow: 6px 6px 12px rgba(0, 0, 0, 0.2);
+      }
+    `}
+`
+/**
+ * Card Component displays content within a card
+ */
+export const Card: React.FC<CardProps> & CardComposition = ({
+  className,
+  image,
+  children,
+  ...rest
+}: CardProps): JSX.Element => {

```

Comment:
You may not need the explicit return value `JSX.Element` here -- `React.FC` already defines that, no?